configure.ac

```diff
@@ -1292,6 +1313,7 @@ echo "  with wallet   = $enable_wallet"
 echo "  with gui / qt = $bitcoin_enable_qt"
 if test x$bitcoin_enable_qt != xno; then
     echo "    qt version  = $bitcoin_qt_got_major_vers"
+    echo "    with bip70  = $enable_bip70"
```

Comment:
Probably would be better to mention "(payment protocol)" here as well.

Reply:
Yep